src/v2/Components/__tests__/Footer.jest.tsx

```diff
@@ -40,4 +41,12 @@ describe("Footer", () => {
     expect(small.find(DownloadAppBadge).length).toEqual(1)
     expect(large.find(DownloadAppBadge).length).toEqual(1)
   })
+
+  it("renders CCPA Wrapper component", () => {
+    const small = getSmallFooterWrapper()
+    const large = getLargeFooterWrapper()
```

Comment:
a small helpful refactor for each of these tests in this file would be to use the jest `beforeEach` hook to assign 
```
small = getSmallFooterWrapper()
large = getLargeFooterWrapper()
```
and then in each test won't need to call `getSmallFooterWrapper` `getLargeFooterWrapper` at all, and instead just reference the variables

Reply:
Oops, I merged before you had a chance to respond to this @The-Beez-Kneez! 😝 Could be a nice follow up!